backend-rust/src/graphql_api.rs

```diff
@@ -787,6 +816,69 @@ pub enum LatestChainParameters {
     ChainParametersV1(ChainParametersV1),
 }
 
+pub struct CurrentChainParameters {
+    reward_period_length:         i64,
+    epoch_duration:               i64,
+    opt_last_payday_block_height: Option<i64>,
+    last_payday_block_slot_time:  chrono::DateTime<Utc>,
+}
+
+#[derive(SimpleObject)]
+#[graphql(complex)]
+pub struct PaydayStatus {
+    next_payday_time:             DateTime,
+    #[graphql(skip)]
+    opt_last_payday_block_height: Option<i64>,
+}
+
+#[ComplexObject]
+impl PaydayStatus {
+    // Future improvement (breaking changes): The front end only uses the
+    // `lastPaydayBlock` which is returned here.
```

Comment:
We should probably create an issue instead of this comment

Reply:
It is already tracked in the issue:
https://github.com/Concordium/concordium-scan/issues/423